format/index.md

```diff
@@ -26,28 +26,38 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 interpreted as described in RFC 2119
 [[RFC2119](http://tools.ietf.org/html/rfc2119)].
 
-## Media Type Negotiation <a href="#media-type-negotiation" id="media-type-negotiation" class="headerlink"></a>
+## Content Negotiation <a href="#content-negotiation" id="content-negotiation" class="headerlink"></a>
 
-Clients **MUST** send all JSON API data in request documents with
-the header `Content-Type: application/vnd.api+json`.
+Clients **MUST** send all JSON API data in request documents with the header
+`Content-Type: application/vnd.api+json`, without any media type parameters.
 
-Clients **MUST** ignore all parameters for the `application/vnd.api+json`
+Clients that include the JSON API media type in their `Accept` header, **MUST**
+specify the media type there at least once without any media type parameters.
+
+Servers **MUST** return a `400 Bad Request` status code if clients violate
+either of the requirements above. The server's response to such violations
+**MUST** contain a JSON API document with an [error object](#error-objects).
+The `"href"` member of that object **MUST** be
+`"http://jsonapi.org/errors/invalid-media-type-parameter"`.
```

Comment:
This is a more specific requirement for errors than anywhere else in the spec. I think we should remove this sentence about error object requirements. I'd prefer to prepare some detailed recommendations for using the error object in different scenarios.